build/build_wikibase.sh

```diff
@@ -26,7 +26,7 @@ GZIP=-9 tar -C "$TEMP_GIT_DIR" -zcf "$TEMP_TAR_DIR"/Wikibase.tar.gz Wikibase
 TARBALL_PATH="$TEMP_TAR_DIR/Wikibase.tar.gz"
 
 if [ -n "$GITHUB_ENV" ]; then
-    echo "TARBALL_PATH="$TARBALL_PATH"" >> $GITHUB_ENV
+    echo "TARBALL_PATH=""$TARBALL_PATH""" >> "$GITHUB_ENV"
```

Comment:
Does this make any difference? `echo "TARBALL_PATH="$TARBALL_PATH""` and `echo "TARBALL_PATH=""$TARBALL_PATH"""` always seem to yield the exact same result.

Reply:
It does but it took me a while to see how. Before this commit $TARBALL_PATH is actually unquoted and followed by a quoted 0 length string.

Given how we know this command is supposed to work it mostly works exactly as planned either way. In this context spaces make no difference (they also just get echoed in like we'd expect).

An edge case where is breaks down is bash expansions. Try `TARBALL_PATH='cat dfs /*'` and you'll see is then spams in the directories of `/`